### PR TITLE
improve laboratory page

### PR DESCRIPTION
**Pull request by B2o5T**
fixes https://github.com/kamilkisiela/graphql-hive/issues/77

https://tortilla-hq.slack.com/archives/C01E8ATBQGN/p1653583554524889

### CODE REVIEW

File: packages/web/app/pages/[orgId]/[projectId]/[targetId]/history.tsx [hunk 1]
```diff
@@ -226,30 +224,30 @@ const Page = ({ versionId }: { versionId: string }) => {
...
         </div>
       </div>
-      <div className="grow">
-        <div className="mb-4 flex items-center justify-between">
+      <div className="flex grow flex-col gap-4">
+        <div className="flex items-center justify-between">
           <Heading>Schema</Heading>
           <ToggleGroup
-            defaultValue="diff"
+            defaultValue="list"
```
Comment (B2o5T) [inline packages/web/app/pages/[orgId]/[projectId]/[targetId]/history.tsx:231]: I guess it was a mistake

File: packages/web/app/pages/[orgId]/[projectId]/[targetId]/laboratory.tsx [hunk 2]
```diff
@@ -1,54 +1,33 @@
...
-  const hasSchemas = schemasQuery.data?.target.latestSchemaVersion?.schemas.nodes.length > 0;
   const endpoint = `${window.location.origin}/api/lab/${router.organizationId}/${router.projectId}/${router.targetId}`;
+
   return (
     <>
       <p className="mb-5 font-light text-gray-500">
         Experiment, mock and create live environment for your schema, without running any backend.
       </p>
-      {schemasQuery.fetching ? (
-        <Spinner className="mt-10" />
```
Comment (B2o5T) [inline packages/web/app/pages/[orgId]/[projectId]/[targetId]/laboratory.tsx:35]: I guess no need to show a spinner, in v1 graphiql showed an error if there is no connected schema

File: packages/web/app/package.json [hunk 3]
```diff
@@ -43,7 +44,7 @@
     "echarts-for-react": "3.0.2",
     "formik": "2.2.9",
     "framer-motion": "4.1.17",
-    "graphiql": "1.4.2",
+    "graphiql": "1.7.2",
```
Comment (B2o5T) [inline packages/web/app/package.json:47]: updated little bit, in latest version `next build` throw an error

**Review by kamilkisiela — APPROVED** [review]
I pushed one fix, because it has to point to `/api/lab/org/project/target` endpoint

**Pull request merged**